packages/tokens/src/fontWeights.ts

```diff
@@ -1,7 +1,18 @@
 export const fontWeights = {
   regular: 400,
   medium: 500,
+  semiBold: 600,
+  /** @deprecated - Prefer {@link semiBold} */
```

Comment:
maybe add a bit more detail on why to prefer `semibold` (and that unless someone's explicitly downloading the `bold` weight, it'll likely default to `semibold`)